### DISCUSSION TITLE
Certifications?

### DISCUSSION THREAD

**Discussion by Yasir-Khan22** [microsoft/Web-Dev-For-Beginners#1001] [General]
I hope all of you are doing well. I wanted to ask a question, will be students given a certification after completing the course.? Furthermore, I have gone through, but I could not find information about it.?

**Comment by ghost** [comment]
yuh,cerificates are there and sorry it does not reflect

**Reply by AsifJishan** [reply]
What do you mean they are there? How do i obtain it?
